test/modes_internal_test.c

```diff
@@ -887,7 +887,6 @@ static int test_gcm128(int idx)
 static void benchmark_gcm128(const unsigned char *K, size_t Klen,
                              const unsigned char *IV, size_t IVlen)
 {
-#ifdef OPENSSL_CPUID_OBJ
```

Comment:
Since you removed the `OPENSSL_CPUID_OBJ` guard, you need to check the return value of `OPENSSL_rdtsc` instead (line 910).

Reply:
For a third time, don't try to "fix" this, removal makes *more* sense.

Reply:
Sorry Andy, I did not intend to ignore https://github.com/openssl/openssl/pull/5231#discussion_r165612740 deliberately. I simply forgot about it. Well then, let's remove the code!